helm/panda/charts/jedi/templates/deployment.yaml

```diff
@@ -15,7 +15,6 @@ spec:
   hostPath:
     path: {{ .Values.persistentvolume.path }}
 ---
-{{- end }}
```

Comment:
I don't quite understand this change. In the production k8s cluster, only admin can create PV but users can still request PVC. The value persistentvolume.create=false is for this configuration. On the other hand, test cluster typically allows even users to create PV. In this case, the value is set to true.

Reply:
I think you are right. I will update it.